x/ecosystem-incentive/spec/02_state.md

```diff
@@ -2,24 +2,24 @@
 
 **NOTE: This is early draft.**
 
-## FrontendIncentive
+## Incentive
 
 ```protobuf
-message FrontendIncentive {
-  string frontend_name = 1;
+message IncentiveStore {
+  string incentive_id = 1;
   repeated string subjects = 2;
   repeated undetermined weights = 3;
 }
 ```
 
-- FrontendIncentive: `"frontend_name" -> format(FrontendIncentive)`
+- Incentive: `"incentive_id" -> format(IncentiveStore)`
 
 ## RewardTable
 
-RewardTable is the record of the rewards for the subject of the `frontend-incentive`.
+RewardTable is the record of the rewards for the subject of the `ecosystem-incentive`.
 
 ```protobuf
-message FrontendReward {
+message Reward {
```

Comment:
@taiki1frsy 
I think the data structure is ambiguous.
Can I see a sample data structure?

Reply:
@mkXultra 
I added description in 02_state.md for more abundant document of the data structures in this module.
Plz refer.